internal/metrics/metrics.go

```diff
@@ -28,29 +26,35 @@ func init() {
 	Meter = provider.Meter("github.com/flipt-io/flipt")
 }
 
-// MustSyncInt64 returns a syncint64 instrument provider based on the global Meter.
+// MustInt64 returns an instrument provider based on the global Meter.
 // The returns provider panics instead of returning an error when it cannot build
 // a required counter, upDownCounter or histogram.
-func MustSyncInt64() MustSyncInt64InstrumentProvider {
-	return mustSyncInt64InstrumentProvider{}
+func MustInt64() MustInt64Meter {
+	return mustInt64Meter{}
 }
 
-// MustSyncInt64InstrumentProvider is a syncint64.InstrumentProvider which panics
-// if it cannot successfully build the requestd counter, upDownCounter or histogram.
-type MustSyncInt64InstrumentProvider interface {
-	// Counter creates an instrument for recording increasing values.
-	Counter(name string, opts ...instrument.Option) syncint64.Counter
-	// UpDownCounter creates an instrument for recording changes of a value.
-	UpDownCounter(name string, opts ...instrument.Option) syncint64.UpDownCounter
-	// Histogram creates an instrument for recording a distribution of values.
-	Histogram(name string, opts ...instrument.Option) syncint64.Histogram
+// MustInt64Meter is a meter/Meter which panics if it cannot successfully build the
+// requestd counter, upDownCounter or histogram.
+type MustInt64Meter interface {
+	// ounter returns a new instrument identified by name and configured
```

Comment:
```suggestion
	// Counter returns a new instrument identified by name and configured
```

Reply:
Thank you. I did a bit of over zealous comment stripping 😂